earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -108,19 +113,21 @@ compare_asset_root_publishvolume()
 check_fusion_master_or_slave()
 {
     if [ -f "$ASSET_ROOT/.config/volumes.xml" ]; then
-        EXISTING_HOST=$(xmllint --xpath "//VolumeDefList/volumedefs/item[1]/host/text()" "$ASSET_ROOT/.config/volumes.xml" | $NEWLINECLEANER)
-
-        case "$EXISTING_HOST" in
-            $HOSTNAME_F|$HOSTNAME_A|$HOSTNAME_S|$HOSTNAME)
-                IS_SLAVE=false
-                ;;
-            *)
-                IS_SLAVE=true
-
-                echo -e "\nThe asset root [$ASSET_ROOT] is owned by another Fusion host:  $EXISTING_HOST"
-                echo -e "Installing $GEEF $GEE_VERSION in Grid Slave mode.\n"
-                ;;
-        esac
+        #TODO: xmllint does not have --xpath option
```

Comment:
We'll need to bring this logic back eventually (I see you've marked it as a TODO already).